css/properties/user-select.json

```diff
@@ -65,6 +68,9 @@
               }
             ],
             "firefox_android": [
+              {
+                "version_added": "69"
```

Comment:
There is no Firefox Android 69.  68 is the last planned version until Fenix is ready for deployment.